custom_components/tuya_local/devices/smartplug_usb_childlock.yaml

```diff
@@ -0,0 +1,95 @@
+name: Plug In WiFi Smart Socket with USB
```

Comment:
Top level name will get prepended all over the UI, so it is better to keep it short eg "Smartplug with USB"